docker-compose.yml

```diff
@@ -53,6 +54,7 @@ services:
       - LOCALSTACK_HOSTNAME=localstack
       - AWS_REGION=eu-west-2
       - AUDIT_QUEUE_URL=http://localstack:4566/audit-events
+      - MY_ONE_LOGIN_USER_ID=andrew
```

Comment:
Is this meant to be included in this PR?

Reply:
Removed, thanks for spotting this.